x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -33,10 +33,10 @@ filebeat.inputs:
   config_version: 2
   interval: 1m
   request.url: https://api.ipify.org/?format=json
-  processors:
-    - decode_json_fields
-        fields: [message]
-        target: json
+processors:
+- decode_json_fields:
+    fields: ["message"]
+    target: "json"
```

Comment:
It is actually [valid](https://www.elastic.co/guide/en/beats/filebeat/7.10/defining-processors.html#where-valid) to define processors at the input level. This doc page is about inputs, so I think we should leave this example in the input.
```suggestion
  processors:
    - decode_json_fields:
        fields: ["message"]
        target: "json"
```

Reply:
Ahh ok! Thank you! Will change it back! @jsoriano Do you know how can I fix the lint error in this PR?

Reply:
It seems `NOTICE.txt` is outdated in master, it should have been updated in https://github.com/elastic/beats/pull/23484, not sure why it was not detected. Opening a PR to fix this: https://github.com/elastic/beats/pull/23549

Reply:
Thank you! It's fixed now.